tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#
+#+language: lg
+#+region: ug
+#+type: literary
+#+contraction: no
+#+grade: 1
+#+dots: 6
+#+direction: both
+#
+# Copyright (C) 2022 Compass Braille, UK, https://compassbraille.org
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# This table was prepared using feedback from Ugandan proof readers of the
+# Luganda Bible 1968 Edition (Bible Society of Uganda, 2013) in Braille.
+#
+# Luganda uses the same 26 latin letters as English plus the letter Ŋŋ (LATIN
+# LETTER ENG: U+014A and U+014B). It uses the same rules as UEB except for
+# this letter. In UEB it is 45-1345. In Luganda it usually appears as a pair
+# but sometimes singly and in both cases it is transcribed as 1345-1245 - the
+# same as the letters "ng". Backtranslation is therefore problematic but there
+# seem to be predicatble patterns as to where and how it is used so that this
+# table works well in both directions in most cases.
```

Comment:
I'm missing references to documentation of the braille code. Does it exist?

Reply:
I don't know that any documentation exists.

Reply:
I will try to find out but communication is difficult.

Reply:
OK thanks.

Reply:
I have not been able to find anything out about this.

Reply:
OK. Could you maybe add a little comments saying that you looked for a formal specification but didn't find one?